Commands/summarize.py

```
@@ -43,83 +34,49 @@ class TranscriptUnavailableError(TranscriptError):
+    headers = {"X-AccessToken": YT_CAPTION_API_TOKEN}
+    params = {
+        "language": language,
+        "include_meta": "false",
+    }
 
-            if not text:
-                raise TranscriptUnavailableError("Transcript returned, but was empty.")
+    try:
+        response = requests.get(endpoint, headers=headers, params=params, timeout=30)
+    except requests.RequestException as e:
+        raise TranscriptError(f"Failed to contact transcript API: {e}") from e
```

Ruff (0.15.5)

[warning] Avoid specifying long messages outside the exception class

(TRY003)